runtime/onert/api/include/nnfw_experimental.h

```diff
@@ -221,13 +221,16 @@ typedef struct nnfw_train_info
 } nnfw_train_info;
 
 /**
- * @brief Set training info
+ * @brief Set training infomation
  * @note  This function should be called after calling {@link nnfw_load_model_from_file}
  *        and before calling {@link nnfw_train_prepare}
  *
+ * @param[in] session The session to be set training infomation
+ * @param[in] info    The training infomation
+ *
  *  @return @c NNFW_STATUS_NO_ERROR If successful
  */
-NNFW_STATUS nnfw_train_set_traininfo(nnfw_session *session, const nnfw_train_info *train_info);
+NNFW_STATUS nnfw_train_set_traininfo(nnfw_session *session, const nnfw_train_info *info);
```

Comment:
not important, just for matching *.cpp argument naming 

https://github.com/Samsung/ONE/blob/64b87a6618e3883e5ff42023de91a3cd7c05cf5d/runtime/onert/api/src/nnfw_api.cc#L391